app-text/texlive/additional-files/createSubpackageInfos.pl

```diff
@@ -0,0 +1,98 @@
+#!/bin/perl
+
+# Helper script to generate the package metadata definitions for the texlive_texmf subpackages
+# usage:
+# 	perl -I $sourceDir/tlpkg createSubpackageInfos.pl <tlpdbroot> > subpackageInfos.recipe
+#
+#	<tlpdbroot> is the root of the TeXLive package database, containing tlpkg/texlive.tlpdb.
+#		It also needs the texmf-dist directory to work.
+#		After these have been symlinked to $sourceDir (by the BUILD function in the recipe),
+#		$sourceDir can be used inside the haikuporter chroot.
+#		Alternatively, after the installation of the texlive_texmf package,
+#		/boot/system/data/texlive can be used.
+#
+# This script uses the TeXLive perl modules, therefore it needs $sourceDir/tlpkg in @INC.
+
+use strict;
+use warnings;
+
+use TeXLive::TLPDB;
+
+print "# This file has been generated by createSubpackageInfos.pl\n\n";
+
+my $tlpdb = TeXLive::TLPDB->new(root => $ARGV[0]);
+my @collections = $tlpdb->collections();
+
+for my $collectionPkgName (@collections) {
+	my $collection = substr $collectionPkgName, 11;
+	my $collectionPkg = $tlpdb->get_package($collectionPkgName);
+	my $shortdesc = $collectionPkg->shortdesc();
+	my $longdesc = $collectionPkg->longdesc();
+	if (length $shortdesc >= 48) { # SUMMARY must not exceed 80 characters in total
```

Comment:
I think we have upgraded to 100 characters now? Or does that not apply to the SUMMARY?

Reply:
That restriction comes from haikuporter, see https://github.com/haikuports/haikuporter/blob/226a85daf6255ad9b8528a349cea5214c36d97eb/HaikuPorter/Port.py#L393

Reply:
That could use an update too I guess on haikuporter side as this has been changed not too long ago in haikuports guidelines: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips

Reply:
On the other hand, if we use SUMMARY in rdef files it would probably look strange in FileTypes?